bin/oneoff/cpa/backfill_user_events.rb

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env ruby
+
+require_relative '../../../deployment'
+require_relative '../../../lib/cdo/shared_constants'
+require 'cdo/db'
+require 'json'
+
+# We cannot require the model easily, so we just repeat the event names
```

Comment:
What if we place the script in the `dashboard/scripts/oneoff`?

Reply:
We have similar backfill scripts already in the `bin/oneoff` path. It was getting "unwieldy" so I, as a compromise, put it in its own directory (then, now, I made that directory `platform` to show team ownership) which is maybe a little silly. But I do not want to hide oneoff scripts in the dashboard path when there is a more-used alternative.